src/amber/router/scope.cr

```diff
@@ -1,9 +1,9 @@
 module Amber
   module Router
     class Scope
-      @stack = [] of String
+      property :stack
 
-      def initialize
+      def initialize(@stack : Array(String) = [] of String)
```

Comment:
Ok for injection, but `@stack : Array(String) = [] of String` should be simplified by

`@stack = [] of String`

Reply:
Thanks for your idea, I modified it in new commit.